packages/sanity/src/structure/panes/document/DocumentPaneProvider.tsx

```diff
@@ -781,24 +786,22 @@ export const DocumentPaneProvider = memo((props: DocumentPaneProviderProps) => {
       timelineStore,
       title,
       value,
-      version: bundlePerspective,
+      selectedVersionName: selectedPerspectiveName,
+      selectedReleaseName,
```

Comment:
Do we need this two as part of the `DocumentPaneProvider` ? Now that the release is always global

Reply:
Don't think so. I'm making a note to remove it. Good call!